custom_components/opensprinkler/__init__.py

```diff
@@ -341,7 +341,6 @@ async def async_will_remove_from_hass(self):
         await super().async_will_remove_from_hass()
         self._coordinator.deregister_time_interval_listener()
 
-    @Throttle(SCAN_INTERVAL)
```

Comment:
Can yo also remove the timedelta import since its not used. and also the file permissions for this file was changed, you might want to check your IDE

Reply:
👍 Addressed both, also removed Throttle from imports